components/common/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+import styles from "./SearchBar.module.scss";
+
+interface SearchBarProps {
+  className?: string;
+  placeholder?: string;
+}
+
+const SearchBar = ({
+  className = "",
+  placeholder = "",
+}: SearchBarProps) => {
+  const [searchQuery, setSearchQuery] = useState("");
+  const router = useRouter();
+
+  const handleSearch = (e: React.FormEvent) => {
```

Comment:
```suggestion
  const handleSubmitSearch = (e: React.FormEvent) => {
```
이렇게 바꿔보는거 어떨까요? 핸들러함수 네이밍 컨벤션 정립이 필요해 보이네요.

Reply:
handleSubmit으로 바꿨습니다